### PR TITLE
Extend Training Operator GH action to also apply a TFJob and assert it runs

### PR DESCRIPTION
**Pull request by NickLoukas**
Refs https://github.com/kubeflow/manifests/issues/2248

### CODE REVIEW

File: .github/workflows/train_operator_kind_test.yaml [hunk 1]
```diff
@@ -29,3 +29,8 @@ jobs:
         kubectl create ns kubeflow
         kustomize build overlays/kubeflow | kubectl apply -f -
         kubectl wait --for=condition=Ready pods --all --all-namespaces --timeout 180s
+
+    - name: Create a TFJob
+      run: |
+        kubectl create -f tests/gh-actions/kf-objects/tfjob.yaml
+        kubectl wait --for=condition=Running tfjobs tfjob-simple -n kubeflow --timeout 300s
```
Comment (kimwnasptd) [inline .github/workflows/train_operator_kind_test.yaml:36]: Doesn't `kubectl wait --for=condition=Running` mean that the script will wait until the TFJob starts running?

We should instead wait until it completes

Comment (NickLoukas) [reply .github/workflows/train_operator_kind_test.yaml]: I changed the condition check to `Succeeded`.

**Comment by NickLoukas** [comment]
Hey @johnugeorge & @andreyvelich! We are trying to use a TFJob in a GH action to test the Training Operator and we're currently using this sample (https://github.com/kubeflow/training-operator/blob/master/examples/tensorflow/simple.yaml). 
The problem is that it take too much time for the TFJob to succeed and we would like to ask if you could provide us with an example that completes as fast as possible.

**Comment by johnugeorge** [comment]
We use this https://github.com/kubeflow/training-operator/blob/master/sdk/python/test/e2e/test_e2e_tfjob.py#L33 in our GHA

**Comment by NickLoukas** [comment]
I've sent a fixup for the commit and changed the TFJob yaml. Now the Job completes successfully.

**Comment by kimwnasptd** [comment]
Thanks @NickLoukas! Can you add a dummy commit here and create a new file to trigger the GH Action?

Just to verify that everything works as expected in the repo as well

**Comment by NickLoukas** [comment]
I added a dummy commit with a test file and everything works as expected here as well.

**Comment by kimwnasptd** [comment]
Looks good! Let's remove this commit and we can merge

**Comment by NickLoukas** [comment]
Just rebased and dropped the test commit.

**Comment by kimwnasptd** [comment]
Thanks for the work!

/lgtm
/approve

**Pull request merged**